docs/cristin.ttl

```diff
@@ -103,22 +112,23 @@
          """ .
 
 :FAGFELT_Class
-    a       :CristinClass;
-    rdfs:comment """The scientific field in which the book is published""" ;
+    a               :CristinClass ;
+    rdfs:comment    """The scientific field in which the book is published""" ;
     rdfs:subClassOf :TYPE_BOK_RAPPORT_Class .
 
 :TYPE_TIDSSKRIFTPUBLIKASJON_Class
-    a       :CristinClass;
-    rdfs:comment """A type publication called tidsskrift""".
+    a            :CristinClass ;
+    rdfs:comment """A type publication called tidsskrift""" .
 
 :TIDSSKRIFT_Class
-    a       :CristinClass;
+    a            :CristinClass ;
     rdfs:comment """UNKNOWN""" .
 
 :VARBEID_EMNEORD_Class
-    a       :CristinClass;
+    a            :CristinClass ;
     rdfs:comment """UNKNOWN""" .
```

Comment:
An array of keywords attached to the publication/result. The keywords have an id, but the list is not actively managed (free to create new ids/keywords)